ui/big_play_button.js

```diff
@@ -31,12 +33,13 @@ shaka.ui.BigPlayButton = class extends shaka.ui.PlayButton {
 
   /** @override */
   updateIcon() {
+    const Icons = shaka.ui.Enums.MaterialDesignSVGIcons;
+    const icon = new shaka.ui.Icon(/* parent= */ null);
     if (this.isEnded() && this.video.duration) {
-      this.button.setAttribute('icon', 'replay');
-    } else if (this.isPaused()) {
-      this.button.setAttribute('icon', 'play');
+      icon.use(Icons.REPLAY);
     } else {
-      this.button.setAttribute('icon', 'pause');
+      icon.use(this.isPaused() ? Icons['PLAY'] : Icons['PAUSE']);
```

Comment:
```suggestion
      icon.use(this.isPaused() ? Icons.PLAY : Icons.PAUSE);
```

Reply:
We can not do this! If we do this, we break the mechanism for customize shaka.ui.Enums.MaterialDesignSVGIcons by the apps, see https://github.com/shaka-project/shaka-player/pull/9115